xapian-letor/api/letor.cc

```diff
@@ -64,8 +64,8 @@ Letor::set_query(const Xapian::Query & query) {
 }
 
 std::vector<Xapian::docid>
-Letor::letor_rank(const Xapian::MSet & mset, Xapian::FeatureList & flist, const char* model_filename) {
-    return internal->letor_rank(mset, flist, model_filename);
+Letor::letor_rank(const Xapian::MSet & mset, const char* model_filename, Xapian::FeatureList & flist) {
```

Comment:
I think we shouldn't load the model during `letor_rank()`. Once we have a REPL on top of this (say, python), I can see people experimenting with queries on top of the same learned model, in which case we'd want to bring up a suitable ranker, load its model, and then run lots of queries without loading the model again. (Loading the model probably isn't hugely slow, but it'll add up, and future rankers might need to load more data, and possibly unpack it or upload onto a CUDA card or something.)